packages/shared/src/components/ProfileMenu.tsx

```diff
@@ -149,7 +149,7 @@ export default function ProfileMenu({
       onClose={onClose}
       closeOutsideClick
       position={InteractivePopupPosition.ProfileMenu}
-      className="w-full max-w-64 !rounded-24 border border-border-subtlest-tertiary"
+      className="!top-14 w-full max-w-64 !rounded-24 border border-border-subtlest-tertiary !bg-accent-pepper-subtlest"
```

Comment:
I'd almost argue it's probably better to make these more dynamic on the InteractivePopup level.
It already has multiple variations there, so feel they should be aligned not overwritten if that makes sense.